ui/src/tasks/containers/TaskEditPage.tsx

```diff
@@ -145,26 +139,12 @@ class TaskEditPage extends PureComponent<
 
     this.props.setTaskOption({key, value})
   }
-
-  private handleChangeTaskOrgID = (orgID: string) => {
-    this.props.setTaskOption({key: 'orgID', value: orgID})
-  }
 }
 
-const mstp = ({
-  tasks,
-  links,
-  orgs,
-}: {
-  tasks: TasksState
-  links: Links
-  orgs: Organization[]
-}): ConnectStateProps => {
+const mstp = ({tasks}: AppState): ConnectStateProps => {
```

Comment:
nit: this ConnectedStateProps shouldnt be necessary because its being passed in as generics to connect on 162